src/Commands/Backup/Automatic/InfoCommand.php

```diff
@@ -28,12 +29,12 @@ class InfoCommand extends TerminusCommand implements SiteAwareInterface
      * @default-string-field weekly_backup_day
      * @return PropertyList
      *
-     * @param string $site_env Site & environment to get the schedule of, in the format `site-name.env`.
+     * @param string $site_env Site & environment in the format `site-name.env`
      *
      * @usage terminus backup:automatic:info <site>.<env>
-     *     Responds with the day of the week backups are scheduled for on the <env> environment of <site>
-     * * @usage terminus backup:automatic:info <site>.<env> --format=table
-     *     Responds with the day of the week and hour of the day backups are scheduled for on <site>.<env>
+     *     Displays when and for how long automatic backups are created and retained.
+     * @usage terminus backup:automatic:info <site>.<env> --format=table
+     *     Displays when and for how long automatic backups are created and retained.
```

Comment:
The text would be correct if the table headers were changed to
`Backups kept for one week are made every day at`
and
`Except for on this day, which is kept for a month`